website/www/site/content/en/get-started/downloads.md

```diff
@@ -96,11 +96,20 @@ versions denoted `0.x.y`.
 
 ## Releases
 
+### 2.63.0 (2025-02-11)
+
+Official [source code download](https://downloads.apache.org/beam/2.63.0/apache-beam-2.63.0-source-release.zip).
+[SHA-512](https://downloads.apache.org/beam/2.63.0/apache-beam-2.63.0-source-release.zip.sha512).
+[signature](https://downloads.apache.org/beam/2.63.0/apache-beam-2.63.0-source-release.zip.asc).
+
+[Release notes](https://github.com/apache/beam/releases/tag/v2.63.0)
+
+
 ### 2.62.0 (2025-01-21)
 
-Official [source code download](https://downloads.apache.org/beam/2.62.0/apache-beam-2.62.0-source-release.zip).
-[SHA-512](https://downloads.apache.org/beam/2.62.0/apache-beam-2.62.0-source-release.zip.sha512).
-[signature](https://downloads.apache.org/beam/2.62.0/apache-beam-2.62.0-source-release.zip.asc).
+Official [source code download](https://archive.apache.org/beam/2.62.0/apache-beam-2.62.0-source-release.zip).
```

Comment:
I think these need to be of the form `https://archive.apache.org/dist/beam/2.62.0/apache-beam-2.62.0-source-release.zip` - right now this link is dead for me, even if I swap in 2.61.0

Reply:
good catch, fixed